medical/views/medical_abstract_entity.xml

```diff
@@ -0,0 +1,404 @@
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                    />
+                                    <field name="street2"
+                                           placeholder="Street 2..."
+                                           class="o_address_street"
+                                           attrs="{'readonly': [('type', '=', 'contact'),
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                    />
+                                    <field name="city"
+                                           placeholder="City"
+                                           class="o_address_city"
+                                           attrs="{'readonly': [('type', '=', 'contact'),
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                    />
+                                    <field name="state_id"
+                                           class="o_address_state"
+                                           placeholder="State"
+                                           options='{"no_open": True}'
+                                           attrs="{'readonly': [('type', '=', 'contact'),
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                           context="{'country_id': country_id, 'zip': zip}"
+                                           domain="[('country_id', '=', country_id')]"
+                                    />
+                                    <field name="zip"
+                                           placeholder="ZIP"
+                                           class="o_address_zip"
+                                           attrs="{'readonly': [('type', '=', 'contact'),
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                    />
+                                    <field name="country_id"
+                                           placeholder="Country"
+                                           class="o_address_country"
+                                           options='{"no_open": True, "no_create": True}'
+                                           attrs="{'readonly': [('type', '=', 'contact'),
+                                                                 ('parent_id', '!=', False)],
+                                                   }"
+                                    />
+                                </div>
+                                <field name="website"
```

Comment:
It seems inappropriate to me on a patient form except for some very special people.